bin/try-handler.js

```diff
@@ -0,0 +1,16 @@
+var handlerModule = require("../dist/bundle.js");
+
+var fakeLambdaContext = {
+  succeed(results) {
+    console.log(results);
```

Comment:
This was a hacky little way to test that the bundle exports the correct function. I'd like to keep it for now, and put more work on this file in another PR.